configuration.md

```diff
@@ -1,11 +1,68 @@
 # Configuration
 
-Environment variables can be used to configure the project:
+The file server-config.json is used to configure the server:
+It should be in the same folder where the  server is started, if it does not exists, default values will be used"
+
+Below is the server-config.json with all default values
+
+````json
```

Comment:
Shouldn't this be another JSON (sub)format that allows these comments?

Reply:
Changed to `json lines` now, was the best available.